src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import { FontIcon, getColor } from '..';
+
+const Bullet = styled.a`
+    color: ${getColor('type')};
+    font-size: ${props => props.theme.variables.headingMediumSize};
+    line-height: ${props => props.theme.variables.headingMediumLineHeight}
+    font-family: ${props => props.theme.variables.mainFont};
+    transition: all 300ms cubic-bezier(0.165, 0.84, 0.44, 1);
+    display: inline-block;
+    cursor: pointer;
+
+    &:hover {
+        color: ${getColor('primary')};
+        transform: translateX(1em);
+    }
+`;
+
+const StyledFontIcon = styled(FontIcon)`
+	margin-right: ${props => props.arrowSpacing};
+`;
+
+const ArrowBullet = props => (
+	<Bullet>
+		<StyledFontIcon
+			arrowSpacing={props.arrowSpacing}
+			name="arrow-alt-right"
+			size={1.2}
+		/>
+		{props.children}
+	</Bullet>
+);
+
+ArrowBullet.defaultProps = {
+	arrowSpacing: '1.2em',
```

Comment:
Consider using horizontalBase instead of hardcoding. (But can you access it from here? Hmmm)

Reply:
I've just removed this prop, instead using horizontalBase directly. If there is a need for different spacing, component will need to be adjusted with a new prop